src/components/books/books.component.js

```diff
@@ -0,0 +1,17 @@
+import bookCover from './../../assets/img/book-cover.png';
+
+const angular = require('angular');
+
+angular.module('app-bootstrap')
+  .component('books', {
+    template: require('./books.html'),
+    controller: ['booksService', function (booksService) {
+      this.bookCover = bookCover;
+      this.books = booksService.getBooksList();
+    }]
+  })
+  .filter('searchBook', function() {
+    return function(items, searchText) {
+      return items.filter(item => searchText ? item.title.toLowerCase().includes(searchText.toLowerCase()) || item.author.toLowerCase().includes(searchText.toLowerCase()) : true);
```

Comment:
You can use an arrow function to make it even shorter 😄 

```suggestion
    return (items, searchText) =>
      items.filter(item => searchText ? item.title.toLowerCase().includes(searchText.toLowerCase()) || item.author.toLowerCase().includes(searchText.toLowerCase()) : true);
```